capstone/src/main/java/com/google/sps/data/SentimentData.java

```diff
@@ -3,15 +3,15 @@
 import com.google.cloud.language.v1.Sentiment;
 import java.io.IOException;
 
-public class SentimentData{
+public class SentimentData {
```

Comment:
I have made edits to this class so that it works on the live server

Reply:
Okay, thanks!